src/common/database.c

```diff
@@ -155,7 +155,8 @@ static gboolean _migrate_schema(dt_database_t *db, int version)
       "raw_auto_bright_threshold REAL, raw_black INTEGER, raw_maximum INTEGER, "
       "caption VARCHAR, description VARCHAR, license VARCHAR, sha1sum CHAR(40), "
       "orientation INTEGER, histogram BLOB, lightmap BLOB, longitude REAL, "
-      "latitude REAL, color_matrix BLOB, colorspace INTEGER, version INTEGER, max_version INTEGER)",
+      "latitude REAL, color_matrix BLOB, colorspace INTEGER, version INTEGER, max_version INTEGER,"
```

Comment:
I don't think this one is correct, and you missed the update in _create_library_schema().

Reply:
You are right, fixing that.